data/migrations/20200625220949_create-profile.js

```diff
@@ -9,11 +9,15 @@ exports.up = (knex) => {
       table.timestamps(true, true);
     })
     .createTable('incidents', (table) => {
-      table.increments();
+      table.string('table_id').notNullable().unique().primary();
```

Comment:
needs to be table.increments('table_id')

Reply:
fixed